vhdbuilder/packer/install-dependencies.sh

```diff
@@ -472,9 +469,9 @@ ls -ltr /usr/local/bin/* >> ${VHD_LOGS_FILEPATH}
 # v1.16.13-hotfix.20200714.1
 # v1.17.7-hotfix.20200714.1
 # v1.17.9-hotfix.20200714.1
-# v1.18.4-hotfix.20200624.1
-# v1.18.6
-PATCHED_HYPERKUBE_IMAGES="
+# v1.18.4-hotfix.20200626.1
+# v1.18.6-hotfix.20200723.1	
+K8S_VERSIONS="
```

Comment:
why do you rename this

Reply:
sorry this was a mistake. Thank you for catching this